npm/opencode-auth/src/plugin.ts

```
@@ -62,13 +84,183 @@ interface TokenFile {
+    try {
+      renameSync(lock.lockPath, lock.cleanupPath);
+      try {
+        rmdirSync(join(lock.cleanupPath, basename(lock.ownerPath)));
+        rmdirSync(lock.cleanupPath);
+      } catch (error) {
+        try {
+          renameSync(lock.cleanupPath, lock.lockPath);
+        } catch {
+          // Preserve the cleanup error below.
+        }
+        throw error;
```

Biome (2.5.3)

[error] Unsafe usage of 'throw'.

(lint/correctness/noUnsafeFinally)